vttools/wrap_lib.py

```diff
@@ -668,7 +668,7 @@ def compute(self):
         ret = library_func(**params_dict)
         if len(output_ports) == 1:
             self.set_output(output_ports[0].name, ret)
-        else:
+        elif ret:
```

Comment:
This line was added to fix the bug when there are no output ports (we discussed about this before).  What do you mean by "scraper"? It happens when the library function does not return anything. In that case, no output will be set.